### PR TITLE
gdk-pixbuf: depends on zstd

### PR DESCRIPTION
**Pull request by Yisus7u7**
Screenshot_20211216-220533

### CODE REVIEW

**Comment by Ladla-saith** [comment]
Plzz give me command 😕

File: packages/gdk-pixbuf/build.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ TERMUX_PKG_VERSION=2.42.6
 TERMUX_PKG_REVISION=2
```
Comment (thunder-coding) [inline packages/gdk-pixbuf/build.sh:6]: Also bump revision

**Review by thunder-coding — CHANGES_REQUESTED** [review]

**Comment by xeffyr** [comment]
@Yisus7u7 That's not the first time when you are ignoring the essential steps on package modifications. Please submit **complete** changes next time!

**Comment by Yisus7u7** [comment]
> @Yisus7u7 That's not the first time when you are ignoring the essential steps on package modifications. Please submit **complete** changes next time!

OK, thanks!

**Pull request merged**